bookstore-actix/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.9"
```

Comment:
We have a top level docker compose containing Postgres only. You can keep this here if you want to start both the server and postgres together.